### ISSUE TITLE
Readme needs more detailed instructions

### ISSUE THREAD

**Issue by doorleyr** [CityScope/CS_cityscopeJS#2]
Here's the email I sent to Aalto as a template:

Get the project from Github—make sure you clone the project rather than downloading the zip file because this project is still in development and you will need to get the updates.

- Open Github Desktop
- Click the ‘+’ icon for adding a new repository
- Under ‘Clone’, search for CS_cityscopeJS and select it
- Run a local server: 
- navigate to the CS_cityscopeJS folder on your local and run a local server (follow these [instructions](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server#Running_a_simple_local_HTTP_server) to run a local server)
- open your web browser and go to http://localhost:8000/index.html
- you should see the web tool in your browser

Experiment with scanning the static image

- enter a table name
- select ‘toggle keystoning’
- select the top left, top right, bottom left and bottom right corners of the image (in that order)
- refresh the page (this is necessary for now because of a bug but this will be fixed in later versions).
- check https://cityio.media.mit.edu/api to see the results

Make updates to the code to work with the Aalto table

- open the main.js file in the javascript folder
- change the gridSize to the size you need. If you have 20x20 bricks and each brick is 4x4 dots, then gridSize=80
- If necessary, change the recognised patterns. To do this you need to change the codes array in the cityIOstruct object. The codes represent white by 1 and black by 0. If you only have lego pieces that are 2x2, then you will need to use codes that are in blocks of 4. eg. a pattern with 4 white dots in the top-left corner and the rest black would be [1,1,0,0,1,1,0,0,0,0,0,0,0,0,0,0]. When you get the 1x1 lego pieces, you can use more complicated patterns.

N.B. whenever you make changes to the project files you will need to save the changes and then clear the cache of your browser. If you don’t do this, your browser will still be using an old version of the project. In most browsers you can use Ctrl + Shift + Del (https://www.zyxware.com/articles/3678/how-to-clear-cache-on-various-browsers-and-their-key-board-shortcuts)

**Comment by agrignard** [comment]
I think is issue raise the fact that the trojan readme might be updated now that some potential external user might want to use this repo

**Comment by agrignard** [comment]
what about a wiki?
https://github.com/CityScope/CS_cityscopeJS/wiki

**Comment by RELNO** [comment]
@agrignard README+tutorial will be done soon, I'm still very much in WIP [but would be glad to see somebody starting it!]
My plan is to do an 'intro' to CSjs for CS team, get feedback, and then write a clear doc.

**Comment by agrignard** [comment]
Yep I can be the beta tester if you want (like playing the dumb guy that need to follow step by step tutorial to make it work)
